operations/network/customer_accessible_networks/Troubleshoot_CMN_Issues.md

```diff
@@ -25,59 +26,60 @@ Example output:
        valid_lft forever preferred_lft forever
 ```
 
-If there is not an address specified, make sure the `can-` values have been defined in `csi config init` input.
+If there is not an address specified, make sure the `cmn-` values have been defined in `csi config init` input.
 
-### Does the NCN have a Default Gateway Configured?
+## Does the NCN have a Default Gateway Configured?
 
-Check the default route on an NCN other than `ncn-m001`. There should be a default route with a gateway matching the can-gateway value.
+Check the default route on an NCN other than `ncn-m001`. There should be a default route with a gateway matching the `cmn-gateway` value.
 
-```screen
+```bash
 ncn-w002# ip route | grep default
```

Comment:
nit: make this copy-paste friendly, remove `ncn-w002#` since this command is agnostic to ncn.

Reply:
Addressed